### DISCUSSION TITLE
Follow up on discussion #220 and how my team is dealing with it

### DISCUSSION THREAD

**Discussion by SDSTony** [executablebooks/meta#242] [Show and tell]
Hi everyone. 

I would like to share how my team is utilizing JupyterBook and GitLocalize to make a documentation on diverse languages. 

A month ago I have posted discussion #220 and the developers of EBP have helped me out on figuring out some solutions. 

However, due to personal time constraints, I was unable to follow up Sphinx documentations from bottom up to find a elegant way to manage one Jupyter Book repo with two different languages. 

Since we are localizing our documentation to only one target language (EN), we decided it is more efficient in time-wise to create another Jupyter Book repo for the target language. 

After we have created a new repo, we have decided to use [GitLocalize](https://gitlocalize.com/) for translating files. It was useful since we could maintain the files on a cloud based system similar to Github. Also it had a feature to make PR to the repo straight away from the GitLocalize site. 

I'm adding the [link to the repo](https://github.com/Pseudo-Lab/Tutorial-Book-en) in case anyone is facing similar problems and need a repo that could be used for bench-marking. 

Thanks~

**Comment by whyjz** [comment]
I posted my method at https://github.com/orgs/executablebooks/discussions/220#discussioncomment-3284310. I use [Weblate](https://hosted.weblate.org/engage/gmt-tutorials/) as the translation platform (free for open-sourced projects), which works well on integrating translated materials into a single repository.

I also adjusted [my website](https://gmt-tutorials.org/en/index.html) to show a dropdown menu for language selection. This is achieved by batch inserting a `<div>` block in the output HTMLs and modifying the CSS settings.
![scr6](https://user-images.githubusercontent.com/19339926/181855891-28d57e2a-8fee-463a-8b09-1451c0af450f.png)

Hope that's helpful too!
